internal/mid/errors.go

```diff
@@ -19,7 +19,11 @@ func ErrorHandler(before web.Handler) web.Handler {
 		ctx, span := trace.StartSpan(ctx, "internal.mid.ErrorHandler")
 		defer span.End()
 
-		v := ctx.Value(web.KeyValues).(*web.Values)
+		// This will cause the service to be shutdown.
```

Comment:
This comment is confusing. It sounds like that will always do the shutdown but it only does if the ctx value is missing.